nextcloud/talk-android
# Restore user status

Resolve #2943

To test this PR:

1. Set a status.
2. Start a call.
3. The status is automatically set to "In a call" when you are in call.
4. You can see the previously set status in the predefined status list and clicking on "Reset status" button should revert the status and close the status dialog.

<img width="321" alt="Screenshot 2024-11-08 at 19 43 30" src="https://github.com/user-attachments/assets/f856cd38-3405-4a81-95ba-f1778b8b0aec">


### 🚧 TODO


### 🏁 Checklist
